sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitCoverageResultParser.java

```diff
@@ -190,6 +190,8 @@ protected void saveCoverageMeasure(FileNode fileNode) {
       context.saveMeasure(phpFile, linesToCoverMetric, totalStatementsCount);
       context.saveMeasure(phpFile, this.uncoveredLinesMetric, uncoveredLines);
       context.saveMeasure(phpFile, this.lineCoverageMetric, ParsingUtils.scaleValue(lineCoverage * 100.0));
+    } else {
+      LOG.info("File's path: {} must be given using absolute path, related coverage data ignored", fileNode.getName());
```

Comment:
Maybe such messages should be "warn"  instead of "info"?

Also IMO such message might be misleading, if report contains path, which simply does not exist. And I feel that this might produce many false-positive messages in case, if report contains coverage results for multiple modules.